### PR TITLE
Aura/bal proper split

### PR DESCRIPTION
**Pull request by SHAKOTN**
## Done:
1. Add additional rebalancing loop in `re_distribute_incentives` function to rebalance bribs into desired split
2. Add new fields to `recon.json` to collect data about split
3. Generate new alloc files with `aura_poc` prefix

### CODE REVIEW

File: fee_allocator/accounting/fee_pipeline.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def run_fees(
     core_pools["polygon"].pop(
         "0x65fe9314be50890fb01457be076fafd05ff32b9a000000000000000000000a96"
     )
+    core_pools["mainnet"].pop(
```
Comment (Tritium-VLK) [inline fee_allocator/accounting/fee_pipeline.py:48]: Can we just remove this from the core pools json in main instead of building in an exception here, or if we are going to build an exception system make it based of lists visable in config/constants?

File: fee_allocator/accounting/fee_pipeline.py [hunk 2]
```diff
@@ -129,11 +132,12 @@ def run_fees(
             Decimal(fee_constants["vebal_share_pct"]),
             aura_vebal_share=Decimal(aura_vebal_share),
         )
-        re_routed_incentives = re_route_incentives(_incentives, chain, reroute_config)
+        # re_routed_incentives = re_route_incentives(_incentives, chain, reroute_config)
```
Comment (Tritium-VLK) [inline fee_allocator/accounting/fee_pipeline.py:135]: Why did we comment this out, if we ever used rerouting wouldn't we want to see it?

File: fee_allocator/helpers.py [hunk 3]
```diff
@@ -334,12 +334,12 @@ def fetch_token_price_balgql_timerange(
         for item in result["tokenGetPriceChartData"]
         if end_date_ts >= item["timestamp"] >= start_date_ts
     ]
-    # if token_addr.lower() == "0x6dbf2155b0636cb3fd5359fccefb8a2c02b6cb51":
-    #     return Decimal(0.39)
-    # if token_addr.lower() == "0x93ef1ea305d11a9b2a3ebb9bb4fcc34695292e7d":
-    #     return Decimal(2300)
-    # if token_addr.lower() == "0xcd5fe23c85820f7b72d0926fc9b05b43e359b7ee":
-    #     return Decimal(2300)
+    if token_addr.lower() == "0x6dbf2155b0636cb3fd5359fccefb8a2c02b6cb51":
```
Comment (Tritium-VLK) [inline fee_allocator/helpers.py:337]: what is all this about?

**Review by Tritium-VLK — COMMENTED** [review]
Few questions/comments in line.

File: fee_allocator/helpers.py (hunk 3 above)
Comment (SHAKOTN) [reply fee_allocator/helpers.py:337]: Missing token prices from balancer subgraph

File: fee_allocator/accounting/fee_pipeline.py (hunk 1 above)
Comment (SHAKOTN) [reply fee_allocator/accounting/fee_pipeline.py:48]: This thing is happening almost every week and it's hard to follow. I will just leave it for now until we fix core pools list

File: fee_allocator/accounting/fee_pipeline.py (hunk 2 above)
Comment (SHAKOTN) [reply fee_allocator/accounting/fee_pipeline.py:135]: moved back

File: fee_allocator/accounting/fee_pipeline.py (hunk 1 above)
Comment (Tritium-VLK) [reply fee_allocator/accounting/fee_pipeline.py:48]: The core pools list should be the thing that chages a lot, not the code.  Did you create that PR at least?

File: fee_allocator/helpers.py (hunk 3 above)
Comment (Tritium-VLK) [reply fee_allocator/helpers.py:337]: Maybe also abstract this to config/constants somewhere?

**Comment by Tritium-VLK** [comment]
https://github.com/BalancerMaxis/multisig-ops/pull/721

File: main.py [hunk 4]
```diff
@@ -15,7 +15,6 @@
 
 
 DELTA = 1000
-# TS_NOW = int(datetime.now().timestamp()) - DELTA
 # TS_NOW = 1703116800
 # TS_2_WEEKS_AGO = 1701907200
 TS_NOW = int(datetime.now().timestamp()) - DELTA
```
Comment (Tritium-VLK) [inline main.py:20]: is this what we want, don't we always want to run at 00:00 GMT on a thursday?

File: fee_allocator/allocations/output_for_msig/2023-12-21_aura_poc.csv [hunk 5]
```diff
@@ -0,0 +1,106 @@
+target,platform,amount
```
Comment (Tritium-VLK) [inline fee_allocator/allocations/output_for_msig/2023-12-21_aura_poc.csv]: shouldn't we be writingb to the main file in the main branch/not poc.  Also can't find teh code anywhere here that changes the filename, but maybe I'm bliind.

**Review by Tritium-VLK — APPROVED** [review]
